code-challenges.js

```diff
@@ -31,24 +21,6 @@ console.log(isThreeAFriend(num3))
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
-//create a function called capIt that takes in an array of strings
-capIt = (array) => {
-
-    let outPutArray= []
-// using for loop to pull one word at a time
-    for(let i=0; i < array.length; i++){
-    // then to split the string into individual letters and put it in an array
-        let wordArr = array[i].split("")
-    // then use the uppercase method to capitalize index 0
-        wordArr[0]= wordArr[0].toUpperCase()
-        let wordJoin = wordArr.join("")
-
-        outPutArray.push(wordJoin)
-    }
-        return outPutArray
-}
-console.log(capIt(randomNouns))
```

Comment:
great! there will be more efficient ways of solving this so make sure you pay attention to the assessment review :)